Tests/MSFT_xWebBindingInformation.Tests.ps1

```diff
@@ -0,0 +1,50 @@
+﻿$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+
+# should check for the server OS
+if($env:APPVEYOR_BUILD_VERSION)
+{
+  Add-WindowsFeature Web-Server -Verbose
+}
+
+Import-Module (Join-Path $here -ChildPath "..\DSCResources\MSFT_xWebsite\MSFT_xWebsite.psm1")
+
+Describe "MSFT_xWebBindingInformation" {
+    It 'Should be able to get xWebsite' -test {
+        # just a good idea.  
+        # I thought it might force the classes to register, but it does not.
+        $resources = Get-DscResource -Name xWebsite
+        $resources.count | should be 1
+    }
+
+    It 'Should compile and run without throwing (But does not currently forcing pass)' -test {

```

Comment:
There is a current issue open for this: https://github.com/PowerShell/xWebAdministration/issues/17